src/Microsoft.ML.Data/Commands/CrossValidationCommand.cs

```diff
@@ -25,14 +25,14 @@ public sealed class CrossValidationCommand : DataCommand.ImplBase<CrossValidatio
         // REVIEW: We need a way to specify different data sets, not just LabeledExamples.
         public sealed class Arguments : DataCommand.ArgumentsBase
         {
-            [Argument(ArgumentType.Multiple, HelpText = "Trainer to use", ShortName = "tr")]
-            public SubComponent<ITrainer, SignatureTrainer> Trainer = new SubComponent<ITrainer, SignatureTrainer>("AveragedPerceptron");
```

Comment:
Note: I am taking option (1) here from https://github.com/dotnet/machinelearning/issues/726

Reply:
Sounds good to me, thanks @eerhardt .

---
In reply to: [212738893](https://github.com/dotnet/machinelearning/pull/729#discussion_r212738893) [](ancestors = 212738893)